apps/public-reference/pages/sign-in.tsx

```diff
@@ -1,13 +1,38 @@
+import { useState } from "react"
 import { useForm } from "react-hook-form"
-import { Button, Field, FormCard, Icon, LinkButton } from "@bloom-housing/ui-components"
+import axios from "axios"
+import {
+  Button,
+  Field,
+  FormCard,
+  Icon,
+  LinkButton,
+  ErrorMessage,
+} from "@bloom-housing/ui-components"
 import FormsLayout from "../layouts/forms"
 
+const apiBase = process.env.listingServiceUrl
+
 export default () => {
   /* Form Handler */
   const { register, handleSubmit, errors } = useForm()
-  const onSubmit = (data) => {
-    console.log(data)
-    alert("Hi " + data.email + "! To be continued...")
+  const [requestError, setRequestError] = useState<string>()
+
+  const onSubmit = async (data: { email: string; password: string }) => {
+    const { email, password } = data
+
+    try {
+      const res = await axios.post(`${apiBase}/auth/login`, { username: email, password })
+      const { accessToken } = res.data
+      console.log(`Got access token '${accessToken}'`)
+    } catch (err) {
+      const { status } = err.response
```

Comment:
This throws an uncaught error when the backend is down, I think because that case leaves `status` undefined. Can we handle that case a little more cleanly?